zone/embparser.cpp

```diff
@@ -1648,6 +1649,7 @@ void PerlembParser::ExportEventVariables(
 			Seperator sep(data);
 			ExportVar(package_name.c_str(), "skill_id", sep.arg[0]);
 			ExportVar(package_name.c_str(), "skill_level", sep.arg[1]);
+			ExportVar(package_name.c_str(), "skill_max", sep.arg[2]);
```

Comment:
Where is the code for this change to export?

Reply:
Oops, that wasn't suppose to be there lol. Removed.